packages/xcm-cfg/src/configs/polkadot/neuroweb.ts

```diff
@@ -0,0 +1,33 @@
+import { AssetRoute, ChainRoutes } from '@galacticcouncil/xcm-core';
+
+import {
+  neuro,
+} from '../../assets';
+import { neuroweb, hydration } from '../../chains';
+import { BalanceBuilder, ExtrinsicBuilder } from '../../builders';
+
+const toHydration: AssetRoute[] = [
+  new AssetRoute({
+    source: {
+      asset: neuro,
+      balance: BalanceBuilder().substrate().system().account(),
+      destinationFee: {
+        balance: BalanceBuilder().substrate().system().account(),
+      },
+    },
+    destination: {
+      chain: hydration,
+      asset: neuro,
+      fee: {
+        amount: 0.044306118,
```

Comment:
not sure where to get this

Reply:
after sending 1 NEURO Neuroweb -> Hydration i have `0.959016`, diff x5 is `0.205`